packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -567,43 +567,30 @@ class _BottomNavigationBarState extends State<BottomNavigationBar> with TickerPr
         break;
     }
     return Semantics(
-      container: true,
       explicitChildNodes: true,
-      child: Stack(
-        children: <Widget>[
-          Positioned.fill(
-            child: Material( // Casts shadow.
-              elevation: 8.0,
-              color: backgroundColor,
+      child: Material(
```

Comment:
@HansMuller Can you review this file since you were involved in the brith of this code? I believe removing the Stacks here should not have any effects on this in 2D, but it does fix the Widget for 3D.

Reply:
I did have to re-generate the the golden files after this change. The difference seems super-minor, though (the icons just shifted by 1px). See diff: https://github.com/flutter/goldens/pull/22/files?short_path=571c691

Reply:
That's concerning. I wonder why anything moved.

Reply:
were you able to track down the cause of this change?

Reply:
I looked into it for a little bit, but couldn't find anything obvious in the render tree.